assets/src/models/routeTab.ts

```diff
@@ -21,16 +32,17 @@ export interface RouteTab {
   saveChangesToTabUuid?: string
 }
 
-export interface RouteTabData {
-  uuid: string
-  preset_name?: string
-  selected_route_ids: RouteId[]
-  ordering: number
-  ladder_directions: LadderDirections
-  ladder_crowding_toggles: LadderCrowdingToggles
-  is_current_tab?: boolean
-  save_changes_to_tab_uuid?: string
-}
+export const RouteTabData = object({
+  uuid: string(),
+  preset_name: nullable(string()),
+  selected_route_ids: array(string()),
+  ordering: nullable(number()),
```

Comment:
question: why is `ordering` wrapped in `nullable` but the old one was not declared as optional?
am I missing something?

Reply:
This is actually another case where Superstruct caught a problem with the original implementation - `ordering` is actually supposed to be nullable, and is optional on `RouteTab`. A null ordering represents a saved preset that isn't currently open, so as soon as I tried loading the application with one of those I ran into an exception from the Superstruct assertion.